app/src/main/java/org/p2p/wallet/home/ui/select/SelectTokenFragment.kt

```diff
@@ -62,4 +67,32 @@ class SelectTokenFragment : BaseFragment(R.layout.fragment_select_token) {
             emptyTextView.isVisible = isEmpty
         }
     }
+
+    override fun onQueryTextSubmit(query: String?): Boolean = false
+
+    override fun onQueryTextChange(newText: String?): Boolean {
+        val searchText = newText.orEmpty()
+        if (searchText.length < QUERY_MIN_LENGTH) {
+            tokenAdapter.setItems(tokens)
+            return true
+        }
+        val filteredItems = tokens.filter {
+            it.tokenName.startsWith(searchText, ignoreCase = true) || it.tokenSymbol == newText
```

Comment:
i think we need to check .contains
tokenName and tokenSymbol .contains newText
to make search more UserFriendly
for now it.tokenSymbol == newText means that only if user will input USDC ... then token will appear